avail/src/main/kotlin/avail/compiler/splitter/Optional.kt

```diff
@@ -77,14 +77,16 @@ import java.util.Collections
  *
  * Construct a new `Optional`.
  *
- * @param positionInName
+ * @param startInName
  *   The position of the group or token in the message name.
```

Comment:
Another place where the parameter name has changed but the comment describing it has not. Is this just a more accurate name or does the comment need to be updated?